### PR TITLE
[azureiothub] Update doc of Azure IoT => disabled since 2.5

### PR DESCRIPTION
**Pull request by flex001**
Following the recommendation of  rlkoshak a note has been added in the documentation page of the Azure IoT Hub connector.
The connector being disabled since 2.5

### CODE REVIEW

File: bundles/org.openhab.io.azureiothub/README.md [hunk 1]
```diff
@@ -28,3 +28,10 @@ The openHAB Azure IoT Hub Connector can operate in 2 modes:
 Publish (only) or publish and command.
 In publish mode, openHAB will sync all its devices and its status changes to Azure.
 In publish & command mode, you can also send cloud to device commands.
+
+::: tip Note
+Some issues have been reported when using this connector.
+Although the devices are successfully created in Azure IoT Hub, updating their state causes an error.
+As of 2.5, this connector has been disabled.
+Work is still in progress and it will be re-enabled in one of the next versions.
```
Comment (J-N-K) [inline bundles/org.openhab.io.azureiothub/README.md]: Is that true? Who is actively working on that?

**Comment by flex001** [comment]
To be honest I’m not 100% sure someone is still working on the Azure IoT Hub connector itself.
I give it a try last week, and I experienced the issue described (switch devices being created, but updating the switch state on/off cause error in the log : constructor with null parameter).

The aim of my change is just to inform other users to not spending time trying with the latest stable 2.5.
Here below a list of related discussions:

Milestone change: https://ci.openhab.org/job/openhab-milestone/8/changes#detail30
Commit of the removal by Kai Kreuzer: https://github.com/openhab/openhab-addons/commit/e43f4525c76b4110361a5a78df3e1c5235713c21
Pull request of removal : https://github.com/openhab/openhab-addons/pull/5944
Complete issue description : https://github.com/openhab/openhab-addons/issues/3647#issue-330968778

Best regards,

Gauthier

**Comment by kaikreuzer** [comment]
I'd suggest to completely remove the code from the repo in this case.
With https://github.com/openhab/openhab-addons/pull/5944, I had actually expected the docs to disappear as well as it does not make sense to have it on the website, while the add-on isn't available.

**Comment by flex001** [comment]
But it is still available for 2.4, isn't it?

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by kaikreuzer** [comment]
The 2.4 documentation is archived at https://www.openhab.org/v2.4/addons/integrations/azureiothub/, this will anyhow not change.

**Comment by kaikreuzer** [comment]
Superseded by https://github.com/openhab/openhab-addons/pull/6807.

**Pull request closed without merging**